editor/.dependency-cruiser.js

```diff
@@ -0,0 +1,371 @@
+module.exports = {
+  forbidden: [
+    /* rules from the 'recommended' preset: */
+    // {
+    //   name: 'no-circular',
+    //   severity: 'warn',
+    //   comment:
+    //     'This dependency is part of a circular relationship. You might want to revise ' +
+    //     'your solution (i.e. use dependency inversion, make sure the modules have a single responsibility) ',
+    //   from: {},
+    //   to: {
+    //     circular: true,
+    //   },
+    // },
+    // {
+    //   name: 'no-orphans',
```

Comment:
no-orphans was problematic because I told dependency-cruiser to discard type-only ts imports. but that bugged out no-orphans as it pointed to some files which contained types only. bummer!

Reply:
How many did this flag up? It might be worth adding exceptions for those files to the config

Reply:
at least two files were false positives